pkg/clustermesh/types/addressing.go

```diff
@@ -218,3 +218,22 @@ func (ac AddrCluster) As20() (ac20 [20]byte) {
 func (ac AddrCluster) AsNetIP() net.IP {
 	return ac.addr.AsSlice()
 }
+
+type PrefixCluster struct {
```

Comment:
This change is part of #22200, right?

Reply:
Ah, yes. During the cherry-picking from my dev branch, I mixed up the commit. I'll rebase this branch after #22200 is merged.

Reply:
Cool, just wanted to point that out, cuz if this commit changes from any edits in #22200 then you'll have to make those edits here too :+1: